### PR TITLE
Added the ability for the bot to "play defence"

### PR DESCRIPTION
**Pull request by dracco1993**
Addresses #19 

There is no logic behind when to play defence yet, but at least the framework is now there for use in the future.

Right now, I just built in a toggle using the "[" key that triggers it.  It's not really too useful with that alone, but will be useful in the future.

### CODE REVIEW

File: bot.user.js [hunk 1]
```diff
@@ -370,13 +377,24 @@ window.onFrameUpdate = function() {
         }
     }
 };
+window.playDefence = function(dir) {
+    window.kd_l = (dir === "l");
+    window.kd_r = (dir === "r");
+    window.setMouseCoordinates(window.getWidth()/2,window.getHeight()/2);
```
Comment (dracco1993) [inline bot.user.js:383]: This sets the mouse to the center of the screen, so we don't move using it.

File: bot.user.js [hunk 2]
```diff
@@ -370,13 +377,24 @@ window.onFrameUpdate = function() {
         }
     }
 };
+window.playDefence = function(dir) {
+    window.kd_l = (dir === "l");
+    window.kd_r = (dir === "r");
```
Comment (dracco1993) [inline bot.user.js:382]: This, as well as [the line above](https://github.com/ErmiyaEskandary/Slither.io-bot/pull/47/files#diff-3deb4bbb2131417b70ba135e5a6f7715R381) simulate the pressing of the left/right arrows, to turn in the smallest radius circle.

File: bot.user.js [hunk 3]
```diff
@@ -402,6 +420,8 @@ window.loop = function() {
                 window.setAcceleration(1);
             }
         }
+        window.kd_l = false;
+        window.kd_r = false;
```
Comment (dracco1993) [inline bot.user.js:424]: And we need to stop pressing the keys, should we want to play normally again.

**Comment by Evanito** [comment]
Tested and works as promised, for accepting this feature I will defer to @ErmiyaEskandary or @FliiFe

**Comment by Evanito** [comment]
Oh, and in the future use the words "Closes #issue" or "Fixes #issue" to have github automatically close that issue when the PR or commit is accepted

**Comment by dracco1993** [comment]
Ah, I couldn't remember what exactly it was.  :smile:  We don't use GitHub issues for work, so I'm out of  practice.

**Comment by ermiyaeskandary** [comment]
What is this supposed to do ?
EDIT - testing...

**Comment by Evanito** [comment]
It is a toggle to spin in a circle to override bot behavior in self defense

PS: Congrats on contributor

**Comment by ermiyaeskandary** [comment]
BTW @dracco1993 you're a contributor now. Welcome to the project !

**Comment by ermiyaeskandary** [comment]
Tested - works

**Pull request merged**